Gemfile

```diff
@@ -47,6 +47,7 @@ gem 'bootsnap', require: false
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
+gem 'rails-controller-testing'
```

Comment:
- [ ] [Optional] Please add the `ffi` gem to your `Gemfile` to help Windows PCs run your tests because this Gem helps solve requiring files problems in Windows.
[Hint] you can add this line to your Gemfile:
```ruby
gem "ffi", "~> 1.15"
```

Reply:
@Omar-Muhamad thanks for your suggestions.I would do so right away

cheers 🥂